### PR TITLE
fix: show columns when unknown column argument [APE-732]

### PR DESCRIPTION
**Pull request by unparalleled-js**
### What I did

Fixes https://github.com/ApeWorX/ape/pull/1277#discussion_r1137889014

### How I did it

Make error message the same.

### Checklist

- [ ] All changes are completed
- [ ] New test cases have been added
- [ ] Documentation has been updated

### CODE REVIEW

File: src/ape/api/query.py [hunk 1]
```diff
@@ -42,14 +42,21 @@ def validate_and_expand_columns(columns: List[str], Model: Type[BaseInterfaceMod
...
         if len(selected_fields) > 0:
             return list(selected_fields)
 
-    raise ValueError(f"No valid fields in {columns}.")
+    err_msg = _unrecognized_columns(deduped_columns, all_columns)
+    raise ValueError(err_msg)
+
+
+def _unrecognized_columns(unknown: Set[str], all_columns: Set[str]) -> str:
+    unrecognized = "', '".join(sorted(unknown - all_columns))
```
Comment (fubuloubu) [inline src/ape/api/query.py:57]:
```suggestion
def _unrecognized_columns(selected_columns: Set[str], all_columns: Set[str]) -> str:
    unrecognized = "', '".join(sorted(selected_columns - all_columns))
```

File: src/ape/contracts/base.py [hunk 2]
```diff
@@ -939,7 +939,6 @@ def __getattr__(self, attr_name: str) -> Any:
             Any: The return value from the contract call, or a transaction receipt.
         """
 
-        handler: Union[ContractEvent, ContractCallHandler, ContractTransactionHandler]
```
Comment (fubuloubu) [inline src/ape/contracts/base.py:942]: Why remove this in this PR?

**Review by fubuloubu — DISMISSED** [review]

File: src/ape/contracts/base.py (hunk 2 above)
Comment (unparalleled-js) [reply src/ape/contracts/base.py:942]: made to separate PR: https://github.com/ApeWorX/ape/pull/1347
can rebase after that merges

Comment (unparalleled-js) [reply src/ape/contracts/base.py]: was part of another PR that merged anyway, but flake8 upgrade pr open nonetheless. resolving convo!

**Review by fubuloubu — APPROVED** [review]

**Pull request merged**